src/cts/src/Clock.cpp

```diff
@@ -95,7 +95,7 @@ Box<double> Clock::computeSinkRegionClustered(
   auto yMin = std::numeric_limits<float>::max();
   auto yMax = std::numeric_limits<float>::lowest();
 
-  for (const auto [sinkX, sinkY] : sinks) {
+  for (const auto& [sinkX, sinkY] : sinks) {
```

Comment:
What's the warning on this one? sinkX and sinkY are floats. They shouldn't need to be references.

Reply:
See discussion in https://github.com/The-OpenROAD-Project/OpenROAD/pull/2751